lexicons/com/atproto/repo/applyWrites.json

```diff
@@ -18,7 +18,8 @@
             },
             "validate": {
               "type": "boolean",
-              "description": "Can be set to 'false' to skip Lexicon schema validation of record data across all operations, 'true' to require it, or leave unset to validate only for known Lexicons."
+              "default": true,
```

Comment:
Don't we still want the semantic of "leave unset to validate only for known Lexicons"? If we add a `default`, we lose that